plugins/inputs/burrow/testdata/error.json

```diff
@@ -0,0 +1,11 @@
+{
+  "error": true,
+  "message": "Detailed error message",
+  "request": {
+    "uri": "/invalid/request",
+    "host": "responding.host.example.com",
+    "cluster": "",
+    "group": "",
+    "topic": ""
+  }
+}
```

Comment:
Can you add a newline at the end of these json files?

Reply:
Done.